### PR TITLE
Added works and about section

### PR DESCRIPTION
**Pull request by zieeco**
## Portfolio Mobile version: Added works and about section

In this project, I;

- Added works and about section
- Styled the works and about section
- Ensured there was no liter error

This is the second milestone of my portfolio project on the mobile version.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello  @zieeco,

You've done pretty well so far! You've followed most of the guidelines. 🎉
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._
- [ ] The decoration on each project should always be background images, not image tags.
- [ ] The project section should be a `ul` tag with `li` tags.

Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -83,6 +83,235 @@ <h6 class="connect">LET'S CONNECT</h6>
...
+        <article class="project-work">
+          <img
+            src="Images/SnapshootPortfolioA.svg"
+            alt="Tonic Snapshoot"
+            class="project-image"
+          />
+          <div class="project-body">
+            <h2 class="project-title">Tonic</h2>
+            <div class="project-info">
+              <span class="project-info-item canopy">CANOPY</span>
```
Comment (mikenath223) [inline index.html:96]: - [ ] The list of projects uses a `ul` tag to group them, and `li` per project. Kindly restructure this to suit the requirement.

File: index.html [hunk 2]
```diff
@@ -83,6 +83,235 @@ <h6 class="connect">LET'S CONNECT</h6>
...
+              <button class="button" type="button">See project</button>
+            </div>
+          </div>
+        </article>
+        <article class="project-work">
+          <img
+            src="Images/SnapshootPortfolioB.svg"
+            alt="Tonic Snapshoot"
+            class="project-image"
+          />
```
Comment (mikenath223) [inline index.html:121]: - [ ] The section has the proper background image (decorations are always background images, not `img` tags). Remove the `img` tags on each project and instead make the images background images.

**Review by george-swift — APPROVED** [review]
Hello Isaac Samuel,

Congratulations! 🎉 Your project is complete! It's time to merge.

Good job implementing the changes requested by the previous reviewer.  

However, there is always room for improvement. Kindly check the comment below.

[OPTIONAL]

- [ ] Your commit messages are descriptive. While that is commendable 👏🏽👏🏽 , it is recommended and generally accepted that git commit messages are always capitalized.

Cheers and Happy coding! 👏🏽👏🏽👏🏽

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**